projects/packages/jetpack-mu-wpcom/src/features/wpcom-comments/wpcom-comments.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Plugin Name: Jetpack MU WPCom Comment Likes
+ * Description: Adds a "Like" action to comment rows and enqueues the necessary assets in the admin area.
```

Comment:
> to comment rows

Nit: I can imagine that a reader without context might be confused by this. Maybe specify with "in WP-Admin's Comments screen"?